addon/components/bs-modal.js

```diff
@@ -407,6 +426,20 @@ export default Ember.Component.extend({
   _observeOpen: Ember.observer('open', observeOpen),
 
   /**
+   * Give the modal (or its autofocus element) focus
+   *
+   * @method takeFocus
+   * @private
+   */
+  takeFocus() {
+    let focusElement = this.$('[autofocus]').first();
```

Comment:
Huh, yeah that's weird, not sure why they are passing in PhantomJS. The functionality works for me because I use the renderInPlace flag to disable the ember-wormhole behavior...